src/MongoDB/Manager.c

```diff
@@ -79,6 +79,8 @@ static bool php_phongo_manager_merge_context_options(zval* zdriverOptions TSRMLS
 		return false;
 	}
 
+	php_error_docref(NULL TSRMLS_CC, E_DEPRECATED, "The \"context\" driver option is deprecated. Please use the corresponding TLS URI options instead.");
```

Comment:
Note that the context's "capath" option is deprecated in favor of the "ca_dir" driver option, which doesn't have a URI option equivalent (along with "crl_file"). I think we can simply omit the second sentence here. If users have specified one or more relevant "context" options (a subset of the [full list of supported SSL context options](https://www.php.net/manual/en/context.ssl.php)), they'll get additional deprecation notices with more specific instructions.